dataGenerator.js

```diff
@@ -186,8 +219,17 @@ for (let index = NUMOFLIBS; index < data.headmaster.length; index++) {
   data.library[randomVillage].headmasterId.push(index);
 }
 
-//Users-----------------------------------------------------
+// Randomly assign remaining tachers to relevant schools
+for (let index = NUMOFLIBS; index < data.teacher.length; index++) {
+  let randomVillage = faker.random.number(NUMOFLIBS);
+  //teacher
+  data.teacher[index].schoolId = randomVillage;
+  //Village - Schools - Librarys
+  data.school[randomVillage].teacherId.push(index);
+  data.school[randomVillage].count_teachers += 1;
+}
 
+//Users-----------------------------------------------------
 const fakeUsers = [
   { id: 0, email: "admin@admin.com", password: "password", role: "admin" },
   { id: 2, email: "bruno@email.com", password: "password", role: "admin" },
```

Comment:
Everything looks good to me, Could you please add a fake user that has a role as "teacher"?
I need to get the account for a teacher to login on the frontend side.